### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by MaksymKos**
[DEMO LINK](https://MaksymKos.github.io/layout_catalog_grid/)

### CODE REVIEW

**Review by etojeDenys — CHANGES_REQUESTED** [review]
let's add auto margin to center these cards 
image

File: src/styles/_catalog.scss [hunk 1]
```diff
@@ -1,11 +1,36 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
+  display: grid;
+  grid-template-columns: repeat(auto-fill);
```
Comment (IvanFesenko) [inline src/styles/_catalog.scss]: 2023-05-15_15h16_28

https://developer.mozilla.org/en-US/docs/Web/CSS/repeat

File: src/styles/_catalog.scss [hunk 2]
```diff
@@ -1,11 +1,36 @@
...
+    & {
+      grid-template-columns: 1fr 1fr 1fr;
+    }
+  }
+
+  @media screen and (min-width: 1024px) {
+    & {
+      grid-template-columns: 1fr 1fr 1fr 1fr;
+    }
+  }
```
Comment (IvanFesenko) [inline src/styles/_catalog.scss]: Use [repeat](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat) as well

File: src/styles/_catalog.scss [hunk 3]
```diff
@@ -1,11 +1,36 @@
...
+
   &__card {
+    box-sizing: border-box;
```
Comment (IvanFesenko) [inline src/styles/_catalog.scss]: box-sizing should be in `*` selector

**Review by IvanFesenko — CHANGES_REQUESTED** [review]

**Review by IvanFesenko — APPROVED** [review]

**Pull request closed without merging**